jobs/bump-lockfile.Jenkinsfile

```diff
@@ -397,14 +392,12 @@ try { lock(resource: "bump-${params.STREAM}") { timeout(time: 120, unit: 'MINUTE
 
     // Destroy the remote sessions. We don't need them anymore
     stage("Destroy Remotes") {
-        parallel aarch64: {
-            pipeutils.withExistingCosaRemoteSession(arch: "aarch64",
-                                                    session: session_aarch64) {
-                shwrap("cosa remote-session destroy")
+        for (arch in archinfo.keySet()) {
+            if (arch == "x86_64") {
```

Comment:
Optional followup: Perhaps worth factoring out `def remoteArches = archinfo.keySet().filter(|| arch != "x86_64")` or so?  (may not be valid groovy syntax)